src/NServiceBus.RavenDB/Shutdown/ShutdownDelegateRegistry.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace NServiceBus.RavenDB.Shutdown
+{
+    using System;
+    using System.Collections.Generic;
+
+    class ShutdownDelegateRegistry : IRegisterShutdownDelegates, IContainShutdownDelegates
```

Comment:
This was intended to spec out a pattern that could be made globally available via Core, but Core ended up going a different direction. Since this pattern is "going nowhere" we need to get rid of it and do the simplest thing that works (just setting up the shutdown in the FeatureStartupTask) which is what we already agreed to change to for NSB.Raven V4.